src/docs/cmdstan-guide/installation.tex

```diff
@@ -454,6 +454,20 @@ \subsection{Downloading and Unpacking \CmdStan Source}
 \end{Verbatim}
 \end{quote}
 
+\subsection{Running \CmdStan Tests}
+
+Google test uses pthread by default. If the library is not available on your
+system, it's still easy to run the tests. Add this to your \code{make/local}
+file:
+\begin{quote}
+\begin{Verbatim}[fontshape=sl,fontsize=\small]
+GTEST_CXXFLAGS += -DGTEST_HAS_PTHREAD=0
```

Comment:
why is this step suddenly necessary? Does Jenkins have to change?

Reply:
Just hunted it down. It has to do with the os_linux in the math library setting it for pretty much every compiler option there. It trickles down from `CC`. Let me figure out a different way around it.